mne/viz/tests/test_evoked.py

```diff
@@ -101,7 +102,7 @@ def test_plot_evoked():
     evoked = epochs.average()
     assert evoked.proj is False
     fig = evoked.plot(proj=True, hline=[1], exclude=[], window_title='foo',
-                      time_unit='s')
+                      time_unit='s', spatial_colors=False)
```

Comment:
Having to pass `spatial_colors=False` to all of these suggests that this is not implemented correctly. Am I right in assuming it's because a warning is emitted if you don't pass `spatial_colors=False`?

To me `spatial_colors='auto'` should mean "use if available", which implies that `False` should be used if they are not, and users should never get a warning in this mode about positions not being available.

Reply:
those were left over from previous PR. Turns out it's *not* because it raises the "no ch locs" warning... it's because of MPL figure introspection stuff that is specific to the tests, which I've now updated.